software/github/datasets/typedb.json

```diff
@@ -1 +1 @@
rocessor/reactive/common/ReactiveIdentifier.java","reasoner/controller/DisjunctionController.java","pattern/equivalence/AlphaEquivalenceTest.java","common/iterator/sorted/SortedIterators.java","pattern/constraint/thing/IsaConstraint.java","logic/resolvable/Concludable.java","BUILD","concept/ConceptManager.java","reasoner/resolution/resolver/ConjunctionResolver.java","reasoner/resolution/resolver/DisjunctionResolver.java","reasoner/processor/AbstractRequest.java","reasoner/processor/InputPort.java","traversal/procedure/CombinationProcedure.java","traversal/procedure/GraphProcedure.java","traversal/scanner/RelationIterator.java",".grabl/automation.yml","reasoner/processor/reactive/RootSink.java","test/integration/reasoner/common/PlannerTest.java","concept/type/impl/RelationTypeImpl.java","reasoner/controller/RootDisjunctionController.java","pattern/equivalence/AlphaEquivalent.java","test/behaviour/reasoner/verification/CompletenessVerifier.java","graph/vertex/Vertex.java","reasoner/controller/ConjunctionController.java","query/QueryManager.java","test/integration/reasoner/BUILD","reasoner/BUILD","reasoner/answer/Mapping.java","database/CoreDatabase.java","common/parameters/Context.java","reasoner/common/Planner.java","reasoner/controller/NegationController.java","reasoner/answer/Explanation.java","concept/type/impl/EntityTypeImpl.java","pattern/BUILD","traversal/procedure/VertexProcedure.java","reasoner/resolution/framework/ResolutionTracer.java","graph/vertex/impl/TypeVertexImpl.java","reasoner/processor/reactive/AbstractStream.java","concept/thing/Thing.java","concept/type/ThingType.java","reasoner/ExplainablesManager.java","reasoner/controller/NestedDisjunctionController.java","reasoner/processor/reactive/Monitor.java","test/integration/reasoner/ReasonerTest.java","traversal/graph/TraversalEdge.java","test/integration/reasoner/resolution/ReiterationTest.java","test/behaviour/concept/type/thingtype/ThingTypeSteps.java","pattern/variable/ThingVariable.java","reasoner/resolution/framework/RequestState.java","reasoner/controller/ConclusionController.java","graph/vertex/ThingVertex.java","test/integration/logic/BUILD","common/exception/TypeDBException.java","common/iterator/sorted/SortedIterator.java","reasoner/resolution/framework/AnswerCache.java","graph/vertex/impl/ThingVertexImpl.java","reasoner/common/Traversal.java","test/integration/traversal/TraversalTest.java","traversal/planner/GraphPlanner.java","reasoner/ExplanationProducer.java","logic/LogicManager.java","reasoner/resolution/resolver/ConcludableResolver.java","server/SessionService.java","reasoner/processor/reactive/common/PublisherRegistry.java","reasoner/ReasonerProducer.java","pattern/constraint/type/ValueTypeConstraint.java","reasoner/controller/RootConjunctionController.java","reasoner/processor/reactive/Source.java","common/iterator/sorted/BaseForwardableIterator.java","test/behaviour/reasoner/verification/CorrectnessVerifier.java","reasoner/processor/reactive/PoolingStream.java"]}
```

Comment:
what's this diff?

Reply:
I've renamed paths involving the directory `.grabl` to `.factory`. We used those files in our example dataset, but I thought it best to exhaustively remove all references to grabl.